### PR TITLE
more CF updates

### PR DESCRIPTION
**Pull request by kalexand-rh**
For after https://github.com/openshift/installer/pull/1794/ merges

@mazzystr, PTAL

### CODE REVIEW

**Review by mazzystr — APPROVED** [review]

**Comment by mazzystr** [comment]
LGTM!  Thank you!

File: modules/installation-creating-aws-worker.adoc [hunk 1]
```diff
@@ -48,8 +54,8 @@ template requires:
     "ParameterValue": "ami-<random_string>" <4>
   },
   {
-    "ParameterKey": "WorkerSubnet", <5>
-    "ParameterValue": "yes" <6>
+    "ParameterKey": "PrivateSubnet", <5>
+    "ParameterValue": "sg-<random_string>" <6>
```
Comment (jianlinliu) [inline modules/installation-creating-aws-worker.adoc]: Copy/paste error? Obviously here should be "Subnet" as "ParameterKey" per CF template change.

Comment (kalexand-rh) [reply modules/installation-creating-aws-worker.adoc]: Thank you, @jianlinliu! The dev source changed again after I edited this file. I am fixing this error.

Comment (jianlinliu) [reply modules/installation-creating-aws-worker.adoc]: A minor suggestion, seen from the "sg-<random_string>", looks like "ParameterValue" is asking some security group name for "Subnet", if possible, pls make the example value for "ParameterValue" more meaningful, but not confuse user.

Comment (kalexand-rh) [reply modules/installation-creating-aws-worker.adoc]: You're right. I changed it to "subnet-<random-string>." Do you see any other changes?

File: modules/installation-creating-aws-worker.adoc [hunk 2]
```diff
@@ -80,8 +86,7 @@ file metadata, which has the format `<cluster-name>-<random-string>`.
 <3> Current {op-system-first} AMI to use for the worker nodes.
 <4> Specify an `AWS::EC2::Image::Id` value.
 <5> A subnet, preferably private, to launch the worker nodes on.
-<6> Specify a subnet from the `PrivateSubnets` value from the output of the
-CloudFormation template for DNS and load balancing.
+<6> Specify a subnet for your VPC.
```
Comment (jianlinliu) [inline modules/installation-creating-aws-worker.adoc]: I think the original description is more clear and reasonable. Why we need such change here?

File: modules/installation-creating-aws-worker.adoc (hunk 1 above)
Comment (jianlinliu) [reply modules/installation-creating-aws-worker.adoc]: lgtm now.

File: modules/installation-creating-aws-worker.adoc (hunk 2 above)
Comment (kalexand-rh) [reply modules/installation-creating-aws-worker.adoc]: Good point. I've restored the original comment.

**Comment by kalexand-rh** [comment]
Merging.

**Pull request merged**